web-client/src/presenter/computeds/TrialSession/getAllIrsPractitionersForSelectHelper.test.ts

```diff
@@ -0,0 +1,58 @@
+import { applicationContextForClient as applicationContext } from '@web-client/test/createClientTestApplicationContext';
+import { getAllIrsPractitionersForSelectHelper as getAllIrsPractitionersForSelectHelperComputed } from './getAllIrsPractitionersForSelectHelper';
+import { runCompute } from '@web-client/presenter/test.cerebral';
+import { withAppContextDecorator } from '@web-client/withAppContext';
+
+describe('getAllIrsPractitionersForSelectHelper', () => {
+  const TEST_IRS_PRACTITIONERS: any[] = [];
+  const getAllIrsPractitionersForSelectHelper = withAppContextDecorator(
+    getAllIrsPractitionersForSelectHelperComputed,
+    {
+      ...applicationContext,
+    },
+  );
+
+  it('should return an object from the helper', () => {
```

Comment:
❓ : Is there a need for this test? It seems the next test is testing for both the "type" of the data returned from the helper and the actual formatted data.

Reply:
My initial test to make sure everything is wired up correctly in the test file. Will remove since there is no benefits.